src/EventStore.Core/TransactionLog/Chunks/TFChunkDb.cs

```diff
@@ -135,16 +183,18 @@ public void Open(bool verifyHash = true, bool readOnly = false)
                         catch (FileBeingDeletedException exc)
                         {
                             Log.Trace("{0} exception was thrown while doing background validation of chunk {1}.",
-                                      exc.GetType().Name, chunk);
+                                exc.GetType().Name, chunk);
                             Log.Trace("That's probably OK, especially if truncation was request at the same time: {0}.",
-                                      exc.Message);
+                                exc.Message);
                         }
                         catch (Exception exc)
                         {
                             var msg = string.Format("Verification of chunk {0} failed, terminating server...", chunk);
                             Log.FatalException(exc, msg);
-                            throw new CorruptDatabaseException(msg, exc);
+                            Application.Exit(ExitCode.Error, msg);
```

Comment:
I've noticed that this line has been reverted here. It's better to keep it this way. The initial plan was to make this testable by catching the exception but right now it cannot be caught from the main thread since it's thrown from a threadpool thread.